### DISCUSSION TITLE
Thank you so much, Josh! ❤️

### DISCUSSION THREAD

**Discussion by tossha** [JoshParnell/ltheory#2] [General]
I really appreciate everything you did, including this effort to release the code. I sincerely hope that you will always love what you do as much as you loved working on LT. Thank you, and I wish you the best! ❤️

**Comment by HHSsosBath** [comment]
Agreed, i read most of the dev logs and watched all of the vlogs, while most of it flew over my head, the passion was truly inspirational. All the best also <3

**Comment by FormalMoss** [comment]
Just played LTP, and got the Easter Egg, and happily quaffed my life for such an enjoyment :)
Amazing Menus that pop open when you click on them, and stay open until you click on them again - SO refreshing!!
Played LTP as per your Youtube Vid, and died like immediately!!  Looking at LT Code on how to enable a cheat-death code! :) :P

**Comment by khlorghaal** [comment]
this is one of the most gorgeous codebases i have ever seen  
no fear of single character variable names, its so concise and quick to read  
if everything looked like this i would actually enjoy programming again

**Comment by RomuLichdevoihr** [comment]
holy Sh*i
its finally here.
i remember laying depressed in my room at my parents place and following every devlog until i finally felt motivated enough to move out and start learnign game development.
now quite a few years later i just lost my job and fiancé and failed to put my own game engine together since i always intended to go inti eventually and only just learned that these repositories are finally released.
if i ever release anythign id definetly put josh in the top section of the credits screen(s?)

i cant thank you enough for holding onto and sharing a spark like this. you just ignited another phenix i guess
-a fan who lurked around for years
